src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,169 @@
+params:
```

Comment:
I haven't reviewed these fully – ideally the descriptions at least should be reviewed by @EoinShaughnessy before shipping.

Reply:
Thanks for the heads up, @36degrees! :)

Since preview is unavailable, I'll copy this content into a Google Doc to get a better idea of what it'll look like.

Reply:
Also, after I've reviewed the descriptions, they'll need to go for 2i (external tech writer review).

Reply:
I'll take your recommendations as they are now @EoinShaughnessy but I'm going to be making some changes to these following Ollie's review. I probably should've said, sorry. Be prepared for these to change a bit.

Reply:
@owenatgov No worries, give me a shout when you need a review!